cmd/juju/application/refresh_test.go

```diff
@@ -822,7 +822,7 @@ func (s *RefreshSuccessStateSuite) TestForcedSeriesUpgrade(c *gc.C) {
 	metadata := strings.Join(
 		[]string{
 			`name: multi-series`,
-			`summary: "That's a dummy charm with multi-series."`,
+			`summary: "That's a dummy charm with series."`,
```

Comment:
☝️ https://github.com/juju/juju/pull/16030#discussion_r1280867979

Reply:
Damn. Oh well, it's a arbitrary string in a test, not worth the effort I think